apps/web/components/booking/BookingListItem.tsx

```diff
@@ -184,6 +196,18 @@ function BookingListItem(booking: BookingItemProps) {
     },
   ];
 
+  const chargeCardActions: ActionType[] = [
+    {
+      id: "charge_card",
+      label: cardCharged ? t("no_show_fee_charged") : t("collect_no_show_fee"),
+      disabled: cardCharged,
+      onClick: () => {
+        setChargeCardDialogIsOpen(true);
+      },
+      icon: FiCreditCard,
+    },
+  ];
+
```

Comment:
Adds the ability to collect the no show fee from Cal rather than going into Stripe

![image](https://user-images.githubusercontent.com/65426560/229405327-7ab036b1-0ca5-42ac-8fbd-6b89c12ef817.png)